Unit-01/03-templating/app.py

```diff
@@ -0,0 +1,35 @@
+from flask import Flask, render_template, url_for, request
+
+app = Flask(__name__)
+
+@app.route("/person/<name>/<age>")
+def show(name, age):
+	return render_template('person-name-age-show.html', name = name, age = age)
+
+@app.route("/calculate")
+def calculate():
+	return render_template('calc.html')
+
+@app.route("/math")
+def math():
+	num1 = float(request.args.get('num1'))
```

Comment:
I think the intent is to do this via route params instead of query args like so:
```python
@app.route("/math/<operation>/<int:num1>/<int:num2>")
```
but your way works as well 😄